File: Unidad_5/C#/Programa4_U5.cs
using System;

class Programa4_U5
{
    static void Main()
    {
        int numero = 24;

        if (EsPrimo(numero))
        {
            Console.WriteLine(numero + " es un número primo.");
        }
        else
        {
            Console.WriteLine(numero + " no es un número primo.");
        }
    }
    static bool EsPrimo(int numero)
    {
        if (numero <= 1)
        {
            return false;
        }
        for (int i = 2; i <= Math.Sqrt(numero); i++)
        {
            if (numero % i == 0)
            {
                return false; 
            }
        }
        return true; 
    }
}

File: Unidad_5/Java/Programa4_U5.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa4_u5;

/**
 *
 * @author leebc
 */
public class Programa4_U5 {

    public static void main(String[] args) {
       
        int numero = 24;

        if (es_Primo(numero)) {
            System.out.println(numero + " es un número primo.");
        } else {
            System.out.println(numero + " no es un número primo.");
        }
    }

    public static boolean es_Primo(int numero) {
        if (numero <= 1) {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(numero); i++) {
            if (numero % i == 0) {
                return false; 
            }
        }
        return true; 
    }
}
